controllers/apply/lib/field-types/checkbox.test.js

```diff
@@ -20,7 +20,7 @@ test('valid field', function () {
 
     field.withValue(['option-1', 'option-2']);
     expect(field.displayValue).toBe('Option 1,\nOption 2');
-    expect(field.validate().error).toBeNull();
+    expect(field.validate().error).toBeUndefined();
```

Comment:
Noisiest non-schema change, the `error` property is now `undefined` instead of `null` if the schema is valid.